supportedBackends/htcondor/src/test/scala/cromwell/backend/impl/htcondor/HtCondorJobExecutionActorSpec.scala

```diff
@@ -386,9 +388,9 @@ class HtCondorJobExecutionActorSpec extends TestKitSuite("HtCondorJobExecutionAc
     val bashScript = Source.fromFile(jobPaths.script.toFile).getLines.mkString
 
     assert(bashScript.contains("docker run -w /workingDir -v"))
-    assert(bashScript.contains(tempDir1.toAbsolutePath.toString))
-    assert(bashScript.contains(tempDir2.toAbsolutePath.toString))
-    assert(bashScript.contains("/call-hello:/outputDir --rm ubuntu/latest echo"))
+    assert(bashScript.contains(tempDir1.toAbsolutePath.toString.md5Sum))
```

Comment:
Oh, ha, I made a new one of these implicits for the EngineJobFileHasher, `EngineJobHashingActor.StringMd5er`... I wonder if we can find a better place for them all